packages/app-runtime/src/AppRuntime.ts

```diff
@@ -300,6 +300,42 @@ export class AppRuntime extends Runtime<AppConfig> {
         await this.lokiConnection.close().catch(logError);
     }
 
+    protected override async startAccounts(): Promise<void> {
```

Comment:
startAccounts should not be defined and called in the runtime, instead please make this private (and remove the override) and then add this:

```ts
public override async start(): Promise<void> {
    await super.start();

    await this.startAccounts();
}
```